### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by anes66** [pltrdgms/plt_illegalsatis#1]
Hello first thank you for sharing this with us but i got a problem when trying to sell item

here are the screenshots

[Error](https://i.imgur.com/iPPNffJ.png)

[Script](https://i.imgur.com/pw7STGz.png)

I didnt change anything in the server.Lua file

**Comment by pltrdgms** [comment]
edit your config. lua 
    ["adrenaline_shot"] = {      -- item db name
db name of the item you want to sell

**Comment by anes66** [comment]
> edit your config. lua
> ["adrenaline_shot"] = { -- item db name
> db name of the item you want to sell

yeah yeah i know that but its not that problem i fixed it myself 

this helped in my case

in the server.Lua Line 18

`if xPlayer.getInventoryItem(exportItemName).count >= amount then`

i changed to 

`if xPlayer.getInventoryItem(exportItemName).count > amount then`
